src/components/ui/UserPicture.vue

```diff
@@ -13,28 +13,9 @@
 </template>
 
 <script>
-import Identicon from 'identicon.js';
-
+import generateIdenticon from '@/assets/js/identicon-avatar';
```

Comment:
`master` doesn't seem to have this file yet, should be part of this PR?

Reply:
:+1: